src/unix/notbsd/mod.rs

```diff
@@ -682,17 +682,33 @@ f! {
         }
     }
 
-    pub fn WIFEXITED(status: ::c_int) -> bool {
-        (status & 0xff) == 0
+    pub fn WIFSTOPPED(status: ::c_int) -> bool {
+        (status & 0x7f) == 0x7f
     }
 
-    pub fn WEXITSTATUS(status: ::c_int) -> ::c_int {
+    pub fn WSTOPSIG(status: ::c_int) -> ::c_int {
         (status >> 8) & 0xff
     }
 
+    pub fn WIFSIGNALED(status: ::c_int) -> bool {
+        (status & 0x7f) + 1 >= 2
```

Comment:
My local definition of this is crazy:

```c
#define __WIFSIGNALED(status) \                     
  (((signed char) (((status) & 0x7f) + 1) >> 1) > 0)
```

I'll... just assume this is the same!